LibreNMS/OS/Rutos2xx.php

```diff
@@ -0,0 +1,52 @@
+<?php
+/**
+ * Rutos2xx.php
+ *
+ * -Description-
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ * @package    LibreNMS
+ * @link       http://librenms.org
+ * @copyright  2019 Jozef Rebjak
+ * @author     Jozef Rebjak <jozefrebjak@icloud.com>
+ */
+
+namespace LibreNMS\OS;
+
+use LibreNMS\Device\WirelessSensor;
+use LibreNMS\Interfaces\Discovery\Sensors\WirelessSnrDiscovery;
+use LibreNMS\Interfaces\Discovery\Sensors\WirelessRssiDiscovery;
+use LibreNMS\OS;
+
+class Rutos2xx extends OS implement WirelessSnrDiscovery,
```

Comment:
Hi
"implements" instead of "implement" is expected at this line :)
Bye

Reply:
@PipoCanaja solved